Dockerfile

```diff
@@ -54,4 +57,4 @@ VOLUME ["/var/atlassian/confluence"]
 WORKDIR ${CONF_HOME}
 
 # Run Atlassian Confluence as a foreground process by default.
-CMD ["/opt/atlassian/confluence/bin/start-confluence.sh", "-fg"]
+CMD ["/opt/atlassian/confluence/bin/start.sh", "-fg"]
```

Comment:
since we're changing this anyways, we perhaps should choose for the more direct script and go by

```
CMD ["/opt/atlassian/confluence/bin/catalina.sh", "run"]
```